utils/keys_test.go

```diff
@@ -252,17 +274,70 @@ func TestImportNoPath(t *testing.T) {
 	b := &pem.Block{
```

Comment:
Totally non-blocking - I think `b` may not be needed anymore?  We just have to read the file into a byte slice, and pass it to the buffer.  The import itself constructs a new PEM block itself - is this pem block used other than for temporary storage?